gm4_shapeless_portals/data/gm4_shapeless_portals/functions/x/flood_fill.mcfunction

```diff
@@ -0,0 +1,20 @@
+# fill new blocks in portal frame
+# @s = portal frame marker
+# at @s
```

Comment:
`at @s` isn't very useful here. Perhaps change it to something like "at a potential new portal block"